src/pal/src/loader/module.cpp

```diff
@@ -1620,6 +1620,8 @@ static HMODULE LOADLoadLibrary(LPCSTR shortAsciiName, BOOL fDynamic)
         shortAsciiName = FREEBSD_LIBC;
 #elif defined(__NetBSD__)
         shortAsciiName = "libc.so";
+#elif !HAVE_GNU_LIBNAMES_H
+        shortAsciiName = "libc.musl-x86_64.so";
```

Comment:
interesting, this is a non-standard setup, but i can confirm that alpine indeed does it this way (they also have the symlink the other way round). so maybe you can add a check for libc.so if libc.musl-x86_64.so.1 is not found, so it works also on void linux/musl and sabotage linux?